packages/create-fuels/src/cli.ts

```diff
@@ -139,18 +139,31 @@ export const runScaffoldCli = async ({
 
   fileCopySpinner.succeed('Copied template files!');
 
+  // Remove typegen files from gitignore
+  const gitignorePath = join(projectPath, '.gitignore');
+  const gitignoreContents = readFileSync(gitignorePath, 'utf-8');
+  const newGitIgnoreContents = gitignoreContents
+    .replace(/src\/sway-api\/contracts\n/g, '')
+    .replace(/src\/sway-api\/predicates\n/g, '')
+    .replace(/src\/sway-api\/scripts\n/g, '')
+    .replace(/src\/sway-api\/index.ts/g, '');
+  writeFileSync(gitignorePath, newGitIgnoreContents);
+
   if (opts.install) {
     const installDepsSpinner = ora({
       text: 'Installing dependencies..',
       color: 'green',
     }).start();
-
     process.chdir(projectPath);
     execSync(packageManager.install, { stdio: verboseEnabled ? 'inherit' : 'pipe' });
-
     installDepsSpinner.succeed('Installed dependencies!');
   }
 
+  // Generate typegen files
+  if (!process.env.VITEST) {
```

Comment:
I have often given this feedback in past PRs - we should _not_ have test-related constants, conditionals, or env vars in the shipped source code. Can we achieve the same without this workaround?

Reply:
@arboleya My bad, I was just using this temporarily locally to quickly debug something and forgot to remove it. I've removed it now

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3175/commits/5fa7fc879060829ec73f82390a9c40524db544ee

Reply:
Resolving as outdated and fixed in above commit.